python/pysrc/kaskada/_execution.py

```diff
@@ -11,19 +11,33 @@
 
 @dataclass
 class _ExecutionOptions:
-    """Execution options passed to the FFI layer.
-
-    Attributes:
-        row_limit: The maximum number of rows to return. If not specified, all rows are returned.
-        max_batch_size: The maximum batch size to use when returning results.
-          If not specified, the default batch size will be used.
-        materialize: If true, the query will be a continuous materialization.
-    """
+    """Execution options passed to the FFI layer."""
 
+    #: The maximum number of rows to return.
+    #: If not specified, all rows are returned.
     row_limit: Optional[int] = None
+
+    #: The maximum batch size to use when returning results.
+    #: If not specified, the default batch size will be used.
     max_batch_size: Optional[int] = None
```

Comment:
Do you want to explain what "batch size in results" indicates here? Or too descriptive for this param

Reply:
This is more of an internal option any way -- it lets us say "when iterating over the 6 elements in this CSV, give me the results in batches of 2 elements". It lets us test the iterator APIs by controlling the batch size coming out.